packages/react/src/examples/forms/validationDynamic.stories.tsx

```diff
@@ -200,45 +130,42 @@ export const Dynamic = () => {
                 optionLabelField="label"
                 options={cities}
                 onChange={(selected: CityOptionType) => {
-                  formik.setFieldValue('city', selected ? selected.label : '');
+                  handleChange('city', selected ? selected.label : '');
```

Comment:
Looks like typing the value for city is broken for both dynamic and hybrid examples 🤔 Clicking on the list works.

![form-validation](https://user-images.githubusercontent.com/66477579/226284677-9f270eae-ccc6-47c8-83ad-31f3d497a93e.gif)

Reply:
Ah true, I might have broken these somewhere in the middle... used to work :D